dict/dict0dict.cc

```diff
@@ -3885,7 +3819,208 @@ void dict_close() {
   mem_free(dict_sys);
   dict_sys = nullptr;
 
-  for (ulint i = 0; i < DICT_INDEX_STAT_MUTEX_SIZE; i++) {
-    mutex_free(&dict_index_stat_mutex[i]);
+  for (auto &i : dict_index_stat_mutex) {
+    mutex_free(&i);
   }
 }
+
+dict_table_lru::dict_table_lru(std::int32_t capacity, std::function<dict_table_t *(std::uint64_t)> table_fetcher)
```

Comment:
My preference is to create a dict namespace and name the class Table_LRU within that.